lib/services/add_product_to_basket.rb

```diff
@@ -0,0 +1,17 @@
+module Shop
+  class AddProductToBasket
+    attr_reader :product_id, :quantity
+
+    def initialize(params)
+      @product_id = params.fetch("product_id").to_i
+      @quantity = params.fetch("quantity").to_i
+    end
+
+    def call
+      BASKET << Basket.new(
+        product_id: product_id,
+        quantity: quantity
+        )
```

Comment:
I use 2-spaces indentation, also automatically corrected by Sublime. Could you provide me with the example how it should like properly?